receiver/elasticapmreceiver/receiver_test.go

```diff
@@ -332,6 +334,42 @@ func TestAgentCfgHandler(t *testing.T) {
 	}
 }
 
+var inputFiles_error = []struct {
```

Comment:
nit: wdyt about moving this variable within the `TestErrors` func?

Reply:
Ok